examples/asio-standalone/main.cpp

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
It seems you've removed the copyright and licence notice from original author in the top of the example. However, the [boost software licence](https://www.boost.org/LICENSE_1_0.txt) requires the Licence to be left in all derived works.

```suggestion
//
// async_tcp_echo_server.cpp
// ~~~~~~~~~~~~~~~~~~~~~~~~~
//
// Copyright (c) 2003-2020 Christopher M. Kohlhoff (chris at kohlhoff dot com)
//
// Distributed under the Boost Software License, Version 1.0.
// (See http://www.boost.org/LICENSE_1_0.txt)
//

```